update-dependencies/update-dependencies.csproj

```diff
@@ -6,7 +6,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.DotNet.VersionTools" Version="1.0.26-prerelease-00708-05" />
+    <PackageReference Include="Microsoft.DotNet.VersionTools" Version="1.0.27-prerelease-01428-01" />
     <PackageReference Include="System.Diagnostics.TextWriterTraceListener" Version="4.0.0" />
+    <PackageReference Include="NuGet.Packaging.Core.Types" Version="4.0.0" />
+    <PackageReference Include="NuGet.Packaging" Version="4.0.0" />
```

Comment:
Looks like the NuGet.Packaging dependency version in the VersionTools nuspec didn't get updated when the one in the VersionTools project.json did (annoying duplication there)... which means this fix makes sense, but it seems like this should really be fixed in BuildTools.